t/mojo/promise.t

```diff
@@ -516,4 +516,29 @@ subtest 'Wait for stopped loop' => sub {
   is_deeply \@results, ['wait'], 'promise resolved';
 };
 
+subtest 'Resolve self' => sub {
+  my $promise = Mojo::Promise->new;
+  my @results = ([], []);
```

Comment:
Other tests don't use `@results` for both.

Reply:
Addressed.